docs/cubeviz/import_data.rst

```diff
@@ -144,3 +144,26 @@ object, you can load it into Imviz as follows::
 
 There is no plan to natively load such objects until ``datamodels``
 is separated out of the ``jwst`` pipeline package.
+
+.. _cubeviz-import-regions-api:
+
+Importing regions via the API
+=============================
+
+If you have a region file supported by :ref:`regions:regions_io`, you
+can load the regions into Cubeviz as follows::
+
+    cubeviz.load_regions_from_file("/path/to/data/myregions.reg")
+
+Unsupported regions will be skipped and trigger a warning. Those that
+failed to load, if any, will be returned as a list of tuples of the
+form ``(region, reason)`` if requested, as follows::
```

Comment:
```suggestion
Unsupported regions will be skipped and trigger a warning. Those that
failed to load, if any, can be returned as a list of tuples of the
form ``(region, reason)``::
```
"if" and "as" appear a lot here. I suggest the same change in `docs/imviz/import_data.rst`.

Reply:
Done. Thanks!